package.json

```diff
@@ -32,8 +32,9 @@
     "@reduxjs/toolkit": "^1.6.2",
     "@testing-library/jest-dom": "^5.15.1",
     "@testing-library/react": "^12.1.2",
+    "@testing-library/react-hooks": "^8.0.0",
```

Comment:
it looks like you don't end up using this (it's commented out in the test)
drop it?

Reply:
yep you're right, thanks for the catch. removed.